earth_enterprise/docs/landing_page/manual/answer/6014171.html

```diff
@@ -17,79 +17,22 @@
 <a name="top_of_file"></a>
 <p><img src="../art/common/logo_col_126x48.png" width="126" height="48" alt="Google logo" /></p>
 <h1><a href="../answer/index.html">Google Earth Enterprise Documentation Home</a> | Developers</h1>
-<h2>Add Python Libraries to GEE Python</h2><p>Python v2.7.5 is bundled with GEE, providing functionality such as publishing and custom search plug-ins.</p>
```

Comment:
what is plan to update these documents ? 
Since mentioning the exact versions here, we would need to update them for every update of python. Can we be generic about versions ?